docs/docs-beta/docs/dagster-plus/getting-started/code-requirements.md

```diff
@@ -1,6 +1,46 @@
 ---
-title: "Code requirements"
-displayed_sidebar: "dagsterPlus"
+title: 'Dagster+ code requirements'
+displayed_sidebar: 'dagsterPlus'
+sidebar_label: "Code requirements"
 ---
 
-# Code requirements
\ No newline at end of file
+Your Dagster project must meet a few requirements to run in Dagster+.
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- A basic understanding of Python project structure and Docker
+- Familiarity with [deploying open source Dagster](/guides/deployment)
```

Comment:
What's the reason for this prereq?

Reply:
Will remove. Not sure if its actually needed.